ibm/service/vpc/resource_ibm_is_instance.go

```diff
@@ -656,6 +656,13 @@ func ResourceIBMISInstance() *schema.Resource {
 										ValidateFunc:  validate.InvokeValidator("ibm_is_virtual_network_interface", "vni_name"),
 										Description:   "The name for this virtual network interface. The name is unique across all virtual network interfaces in the VPC.",
 									},
+									"protocol_state_filtering_mode": {
```

Comment:
missing  data source and data source test changes

Reply:
only the VNI and instance template API response contains protocol_state_filtering_mode. So only the data sources for VNI and VNI collection and template has the changes.
Test cases are added